executor/aggfuncs/row_number_test.go

```diff
@@ -1,3 +1,16 @@
+// Copyright 2019 PingCAP, Inc.
```

Comment:
2019 - > 2020

Reply:
In addition to this file, should i fix other files?

Reply:
IMO, just fix this file is ok.

Reply:
got it.